### PR TITLE
Hovering a button should change the cursor to pointer unless disabled

### PR DESCRIPTION
**Pull request by yuki24**
This is one of the QA items in https://github.com/artsy/collector-experience/issues/228

### CODE REVIEW

File: src/components/buttons/facebook.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default styled(FacebookButton)`
   flex-direction: row;
 
   &:hover:not(:disabled) {
+    cursor: pointer;
```
Comment (l2succes) [inline src/components/buttons/facebook.tsx]: You only need to put it in the Base Button as all the other Button components extend from it.

Comment (yuki24) [inline src/components/buttons/facebook.tsx]: 👍

**Comment by mzikherman** [comment]
Needs a rebase but looks good!

**Comment by yuki24** [comment]
rebased!

**Pull request merged**